apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/OpenApiDocs.java

```diff
@@ -242,6 +244,30 @@ private void addPostSpecifications(Map<String, Object> spec, String apiVersion)
     }
   }
 
+  /**
+   * Adds deprecated flag to certain HAPI FHIR generated get endpoints. Since we are passed the
+   * entire YAML tree, we can traverse it until we get to the needed get node to add the flag.
+   *
+   * @param spec the OpenAPI spec Map to update with deprecated info.
+   * @param apiVersion either V1 or V2
+   */
+  private void addDeprecatedFlag(Map<String, Object> spec, String apiVersion) {
+
+    // iterate over paths and add post specification to _search endpoints
+    var paths = (Map<String, Object>) spec.get("paths");
+    for (String path : paths.keySet()) {
+      if ((apiVersion.equals(API_VERSION_2)
+              && (path.endsWith("ExplanationOfBenefit/_search")
```

Comment:
I think we also want to include the search endpoints that don't have the `/_search` segment at the end.